docs/interpreter/livy.md

```diff
@@ -51,11 +51,6 @@ Example: `spark.master` to `livy.spark.master`
     <th>Description</th>
   </tr>
   <tr>
-      <td>livy.spark.master</td>
-      <td>local[*]</td>
-      <td>Spark master uri. ex) spark://masterhost:7077</td>
-    </tr>
-  <tr>
     <td>zeppelin.livy.url</td>
```

Comment:
There is still `livy.spark.master` mention under this section: http://zeppelin.apache.org/docs/0.7.0-SNAPSHOT/interpreter/livy.html#configuration Seems this needs to be updated accordingly.